spyder/plugins/breakpoints/plugin.py

```diff
@@ -113,7 +113,7 @@ def on_initialize(self):
         self.create_action(
             BreakpointsActions.ListBreakpoints,
             _("List breakpoints"),
-            triggered=lambda: self.switch_to_plugin(),
```

Comment:
That doesn't work because `sig_switch_to_plugin_requested` is connected to `MainWindow.switch_to_plugin` not to `SpyderDockablePlugin.switch_to_plugin`